.golangci.yml

```diff
@@ -38,10 +38,17 @@ linters:
 issues:
   max-same-issues: 0
   max-issues-per-linter: 0
+  exclude-use-default: false
   # List of regexps of issue texts to exclude, empty list by default.
   exclude:
     - Using the variable on range scope `(tc)|(rt)|(tt)|(test)|(testcase)|(testCase)` in function literal
     - "G108: Profiling endpoint is automatically exposed on /debug/pprof"
+    - Error return value of .((os\.)?std(out|err)\..*|.*Close|.*Flush|os\.Remove(All)?|.*print(f|ln)?|os\.(Un)?Setenv). is not checked
+    - (comment on exported (method|function|type|const)|should have( a package)? comment|comment should be of the form)
+    - Subprocess launch(ed with variable|ing should be audited)
+    - (G104|G307)
+    - (Expect directory permissions to be 0750 or less|Expect file permissions to be 0600 or less)
+    - package comment should be of the form "Package (.+) ..."
```

Comment:
I would suggest to restructure this a bit so we group
- Default golangci exclusions that we would like to remove as soon as possible
- Default golangci exclusions that we preserve (possibily explaining why)
- Additional exclusions that we added in CAPI (possibily explaining why)

Also, in the [golangci doc](https://golangci-lint.run/usage/configuration/) I see exclusions being numbered; what about preserving this number inner comments ?